### DISCUSSION TITLE
想要个Table 综合演示例子

### DISCUSSION THREAD

**Discussion by m916423330** [dotnetcore/BootstrapBlazor#321] [Q&A] [answered]
1.导入导出
2.分页功能
3.增删改查
4.批量删除
5.批量编辑(审核)
6.列排序与列搜索
7.顶部搜索实现所有列搜索
8.高级搜索实现多条件搜索
9.顶部与刷新与视图列
10.实现文本类型明细行
11.列的统计
12.隐藏列,时间日期列格式化

**Comment by densen2014** [comment, marked as answer]
https://github.com/dotnetcore/BootstrapBlazor/discussions/322
